CONTRIBUTING/README.md

```diff
@@ -101,6 +101,7 @@ $ git add <updated files>
 $ git commit -m "Meaningful message for the update"
 # pull new changes
 $ git checkout master
+$ git fetch upstream
```

Comment:
Don't think upstream is required here, only `git fetch` would suffice.

Reply:
![image](https://github.com/user-attachments/assets/0f175210-8fcf-4567-baae-43299da4d396)

Reply:
I believe `git fetch` by default tries to fetch from origin.